src/backend/base/langflow/components/jigsawstack/vocr.py

```
@@ -0,0 +1,107 @@
+                    params["prompt"] = self.prompt
+                elif isinstance(self.prompt, str):
+                    params["prompt"] = [self.prompt]
+                else:
+                    raise ValueError("Prompt must be a list of strings")
+            if self.url:
+                params["url"] = self.url
+            if self.file_store_key:
+                params["file_store_key"] = self.file_store_key
+
+            if self.page_range_start and self.page_range_end:
+                
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)